manifests/tftp/copy_file.pp

```diff
@@ -0,0 +1,11 @@
+# Copy a TFTP file
+define foreman_proxy::tftp::copy_file(
+  $target_path,
+  $source_file = $title,
+) {
+  $filename = inline_template('<%= File.basename(source_file) %>')
```

Comment:
This PR is not really relevant before jessie is released, so maybe there will a new stdlib release until then.